src/main/kotlin/org/neo4j/graphql/handler/QueryHandler.kt

```diff
@@ -97,7 +97,7 @@ class QueryHandler private constructor(
         } else {
             "($variable:${label()})"
         }
-        val where = where(variable, fieldDefinition, type, propertyArguments(field), field)
+        val where = where(variable, fieldDefinition, type, propertyArguments(field), field, env)
```

Comment:
I don't think we should pass `env` through to an inner method but just the parameters/variables